core-services/importExport/src/main/java/org/silverpeas/core/importexport/versioning/VersioningImport.java

```diff
@@ -68,11 +69,12 @@ public VersioningImport(UserDetail user) {
     this.user = user;
   }
 
-  public int importDocuments(String objectId, String componentId, List<AttachmentDetail> attachments,
+  public List<Pair<AttachmentDetail, SimpleDocument>> importDocuments(String objectId,
+      String componentId, List<AttachmentDetail> attachments,
```

Comment:
See my previous comment about returning a pair of `AttachmentDetail` and `SimpleDocument` instead of a `SimpleDocument` (which embeds the actual saved `AttachmentDetail`)